### PR TITLE
[keras/applications/efficientnet.py]  Standardise docstring usage of "Default to"

### PR DESCRIPTION
**Pull request by SamuelMarks**
This is one of many PRs. Discussion + request to split into multiple PRs @ #17748

### CODE REVIEW

File: keras/applications/convnext.py [hunk 1]
```diff
@@ -754,10 +754,10 @@ def preprocess_input(x, data_format=None):
 
     Args:
       x: A floating point `numpy.array` or a `tf.Tensor`.
-      data_format: Optional data format of the image tensor/array. Defaults to
-        None, in which case the global setting
-        `tf.keras.backend.image_data_format()` is used (unless you changed it,
-        it defaults to "channels_last").{mode}
+      data_format: Optional data format of the image tensor/array. `None` means
+        the global setting `tf.keras.backend.image_data_format()` is used
+        (unless you changed it, it uses "channels_last").{mode}.
```
Comment (qlzh727) [inline keras/applications/convnext.py:759]: I think the {mode} here is a place holder, which need to be appear at the end of the line.

File: keras/applications/efficientnet.py [hunk 2]
```diff
@@ -852,10 +852,10 @@ def preprocess_input(x, data_format=None):
 
     Args:
       x: A floating point `numpy.array` or a `tf.Tensor`.
-      data_format: Optional data format of the image tensor/array. Defaults to
-        None, in which case the global setting
-        `tf.keras.backend.image_data_format()` is used (unless you changed it,
-        it defaults to "channels_last").{mode}
+      data_format: Optional data format of the image tensor/array. `None` means
+        the global setting `tf.keras.backend.image_data_format()` is used
+        (unless you changed it, it uses "channels_last").{mode}.
```
Comment (qlzh727) [inline keras/applications/efficientnet.py:857]: Same here.

**Review by qlzh727 — CHANGES_REQUESTED** [review]

File: keras/applications/efficientnet.py (hunk 2 above)
Comment (SamuelMarks) [reply keras/applications/efficientnet.py:857]: @qlzh727 The issue is that `Defaults to` also needs to appear at the end of the string. Actually are you sure `{mode}` is only replaced when it appears at the end of the string? - Where does this happen? - Happy to modify that interpolation code.

Comment (qlzh727) [reply keras/applications/efficientnet.py:857]: Well. I think they are a typo here then. We can remove the {mode}.

https://www.tensorflow.org/api_docs/python/tf/keras/applications/convnext/preprocess_input shows that the {mode} is not replaced properly.

It was replaced in some other application models, eg https://github.com/keras-team/keras/blob/901950201d867c85ec34f4d0c9201aea2c15a65d/keras/applications/imagenet_utils.py#L126

**Comment by haifeng-jin** [comment]
Pending on author's change.

**Comment by SamuelMarks** [comment]
@haifeng-jin Fixed

File: keras/applications/convnext.py (hunk 1 above)
Comment (qlzh727) [reply keras/applications/convnext.py:759]: I think the {mode} here can be removed.

**Review by haifeng-jin — COMMENTED** [review]
LGTM. No more comments from me.

**Pull request merged**